js/layout-javascript/small-card-code.js

```diff
@@ -732,6 +738,16 @@ DynamicList.prototype.prepareData = function(records) {
     records = prefiltered;
   }
 
+  // Add flag for likes
+  records.forEach(function(obj, i) {
+    // Add bookmarks flag
+    if (_this.data.social && _this.data.social.bookmark) {
+      records[i].bookmarksEnabled = true;
+    } else {
+      records[i].bookmarksEnabled = false;
+    }
```

Comment:
`records[i].bookmarksEnabled = _this.data.social && _this.data.social.bookmark`

...should be able to replace these 4 lines